src/main/java/core/basesyntax/Draw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Draw {
+    void drawFigure(Figure randomFigure);
```

Comment:
The draw method must have no input parameters
```suggestion
public interface Drawable {
    void draw();
```